tasker/job_template/generate.py

```diff
@@ -12,6 +13,8 @@ def generate_tasks(template_id):
         rep *= 30
     start = template.starting_date
     timestamp = datetime.datetime.fromtimestamp(start)
+    timestamp = timestamp.replace(hour=template.hour, minute=0)
+    timestamp = timestamp.astimezone(pytz.timezone(user.timezone))
```

Comment:
I think the timezone belongs as a parameter to `fromtimestamp` above.  See [this](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromtimestamp).

Reply:
Updated and tested.